.github/workflows/if-nodejs-version-bump.yml

```diff
@@ -35,6 +35,13 @@ jobs:
           node-version: "${{ steps.lockversion.outputs.version }}"
           cache: 'npm'
           cache-dependency-path: '**/package-lock.json'
+      - if: steps.packagejson.outputs.exists == 'true'
+        name: Setup Node.js
```

Comment:
This should not be changed.

Reply:
I've spend time trying to figure out which merge was wrong, but couldn't replicate it.
I manually checked the code and I came up with the following PR that need to be merged to fix this:

- https://github.com/asyncapi/parser-js/pull/802
- https://github.com/asyncapi/parser-js/pull/804